crud/select/iterator.lua

```diff
@@ -34,7 +33,8 @@ function Iterator.new(opts)
         iteration_func = opts.iteration_func,
 
         plan = opts.plan,
-        after_tuple = opts.after_tuple,
+        reversed = opts.plan.reversed,
```

Comment:
I could consider this change only as attempt to make me resolve merge conflicts at [merger integration](https://github.com/tarantool/crud/pull/38) branch and finally put "reversed" this logic back to select.

I suggest you remove this change and just check `opts.first < 0` just at select. Because it's actually unused in all places except select.

Reply:
OK, I was thinking about checking `opts.first < 0` in select, geniuses' thoughts converge.